docs/TEST.md

```diff
@@ -16,6 +16,14 @@ So, for a reliable run, that also includes logs for failed tests, run:
 ./scripts/test.ps1 -LogProfile Full.Light
 ```
 
+Note that in windows you will need to use Powershell 7 to create the build and Powershell 7 (in administrator mode) to run the tests.
```

Comment:
On Windows, I believe you only need Admin to get logs, not actually run the tests. Also, since powershell was already specified as a requirement, I don't think you need to state it again here.

Reply:
Just tested. Yes, we only need Admin to get the logs. Updated the statement. If it still seems redundant I can remove it.